### ISSUE TITLE
Change MinimumOSVersion to 8.0 to support Carthage builds

### ISSUE THREAD

**Issue by lvillani** [Alamofire/Alamofire#352]
At the moment Carthage can build Alamofire for iOS but since it targets iOS 7, the resulting application cannot be submitted to iTunes Connect.

I already opened a bug report on Carthage to ask for the best course of action (https://github.com/Carthage/Carthage/issues/351), they suggested to open a bug report here.

I see that some bug reports have already been closed in the past because Alamofire needs to support iOS 7 (Is the Xcode project configured that way in order to have Travis build and test against that version of iOS?).

At this point I'm not sure how to proceed.

**Comment by pseudovirtual** [comment]
A fresh checkout and build in Xcode produces this warning
```ld: warning: embedded dylibs/frameworks only run on iOS 8 or later```
Changing the deployment target from 7.0 to 8.0 solves the warning.
There are multiple reports that this warning causes rejection during App Store submission, *even though* it will mostly run fine on Simulator / devices.

**Comment by cnoon** [comment]
I am seeing the same behavior. 

It seems to me that the best course of action would be to have the deployment target set to 8.0 in the project. That will solve the problem for Carthage and won't affect CocoaPods. 

For users that are deploying their app with a deployment target of 7.0, they cannot submit to Apple using Frameworks anyways. They will need to pull in the Alamofire.swift file manually into their project. This approach doesn't care what the deployment target is set to in the `Alamofire iOS` target because they are manually copying the file into their project.

Maybe there's another issue at hand that I'm not aware of at the moment (please comment if so), but it seems to me that the iOS deployment target in the project should be set to 8.0 to solve all the use cases for deployment.

**Comment by nvh** [comment]
:+1:

**Comment by bassrock** [comment]
:+1:

**Comment by bassrock** [comment]
While you may get a warning. You do indeed get a submission error:

```
ERROR ITMS-90362: "Invalid Info.plist value. The value for the key 'MinimumOSVersion' in bundle/Frameworks/Alamofire.framework is invalid. The minimum value is 8.0"
```

**Comment by lvillani** [comment]
@mattt Any opinion on this issue? I could make a PR bumping the minimum OS version requirements if you want.

**Comment by lvillani** [comment]
Oh wait, there's already a PR open: #375

**Comment by mattt** [comment]
Fixed by #400 & #401
